sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/schema.yaml

```diff
@@ -1,5 +1,11 @@
 fields:
 
+- mode: NULLABLE
+  name: submission_date
```

Comment:
can we just partition on date?

Reply:
unfortunately not because `date` and Airflow's execution date will not match.